Doxyfile

```diff
@@ -0,0 +1,22 @@
+PROJECT_NAME           = "OpenROAD's Doxygen Test"	
+PROJECT_BRIEF          = "This is just a test of Doxygen"
+PROJECT_LOGO           = ""
+OUTPUT_DIRECTORY       = "./_readthedocs/html/doxygen_output"
+DISTRIBUTE_GROUP_DOC   = YES
+EXTRACT_ALL            = YES	
+EXTRACT_PRIVATE        = YES
+EXTRACT_PACKAGE        = YES
+EXTRACT_STATIC         = YES
+EXTRACT_ANON_NSPACES   = YES
+INPUT                  = ./src/odb/include/odb/db.h ./docs/doxymain.md
```

Comment:
Do we need to set the input manually? My hope was to have it scan all source files under `src/`

Reply:
Yes, otherwise with a recursive file search the build will fail with too high runtime (> 30 mins). 

https://readthedocs.org/projects/openroad/builds/21726084/